### ISSUE TITLE
Does this resolve the refs to other schemas?

### ISSUE THREAD

**Issue by ppazos** [goodeggs/merge-json-schemas#1]

**Comment by serhalp** [comment]
Yes and no. This is a very simple module that only does what it says on the tin:
- merge properties across all schemas
- error if properties are incompatible (e.g. `foo` is a string in one schema but an integer in another)
- create a union of "required" object properties at some object path

So, no, there is no logic for resolving refs. But refs should still work for most use cases, since two refs will be considered to "match" and be merged successfully. However, two different refs that ultimately resolve to compatible schemas or schemas whose required object keys (at any level of nesting) must be merged – this will not work, and the module will fail because the refs are different (I believe). Feel free to try writing some failing tests to confirm.
